app/code/Magento/Paypal/Controller/Express/AbstractExpress/PlaceOrder.php

```diff
@@ -111,6 +117,10 @@ public function execute()
                     ->setLastRealOrderId($order->getIncrementId())
                     ->setLastOrderStatus($order->getStatus());
             }
+            if ($order->hasInvoices() > 0 && $order->getStatus() == 'processing') {
```

Comment:
The `hasInvoices` method returns boolean value. We don't need to compare it with any integer values.
Please also use strict comparison for string values and try to use constants whenever it's possible. E.g. `\Magento\Sales\Model\Order::STATE_PROCESSING` in this particular case.

So I would change the condition in the following way:
```
if ($order->hasInvoices() && $order->getStatus() === Order::STATE_PROCESSING) {
```